### PR TITLE
Added note on weight decay for tf.contrib.opt optimizers.

### PR DESCRIPTION
**Pull request by Ouwen**
Addresses: #22763

### CODE REVIEW

**Review by alextp — CHANGES_REQUESTED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @akshaym: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by alextp — APPROVED** [review]

**Comment by tensorflowbutler** [comment]
Nagging Assignee @akshaym: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
